scripts/update-modules.sh

```diff
@@ -36,7 +36,7 @@ for MODULE in "OPENWRT" ${GLUON_FEEDS}; do
 	}
 
 	# fetch the commit id for the HEAD of the module
-	REMOTE_HEAD=$(git ls-remote "${REMOTE_URL}" "${REMOTE_BRANCH}" | awk '{ print $1 }')
+	REMOTE_HEAD=$(git ls-remote "${REMOTE_URL}" "refs/heads/${REMOTE_BRANCH}" | awk '{ print $1 }')
```

Comment:
Let's do it the git way instead.

```suggestion
	REMOTE_HEAD=$(git ls-remote --heads "${REMOTE_URL}" "${REMOTE_BRANCH}" | awk '{ print $1 }')
```

Reply:
Nice. Wasn't aware of that :)